examples/fastify-http-server-ts/src/index.ts

```diff
@@ -0,0 +1,37 @@
+import fastifyPlugin from 'fastify-plugin';
+import * as store from '@leapfrogtechnology/async-store';
+import Fastify, { FastifyInstance, FastifyReply, FastifyRequest } from 'fastify';
+
+import { storeParamsPlugin, calculateSum } from './plugin';
+
+const fastifyServer: FastifyInstance = Fastify({ logger: true });
+
+fastifyServer.register(fastifyPlugin(store.initializeFastifyPlugin()));
+fastifyServer.register(fastifyPlugin(storeParamsPlugin));
+
+fastifyServer.register((fastifyInstance, opts, done) => {
+  fastifyInstance.register(fastifyPlugin(calculateSum));
+  fastifyInstance.get('/', (req: FastifyRequest, reply: FastifyReply) => {
+    const a = store.get('a');
+    const b = store.get('b');
+    const sum = store.get('sum');
+
+    reply.send(`Sum of ${a}, ${b}: ${sum}\n`);
+
+    reply.log.info('Response Sent');
+  });
+
+  done();
+});
+
+const start = async () => {
```

Comment:
Let's use this syntax for consistency.

```suggestion
async function start() {
```